MODULE_EVALUATION_TEMPLATE.MD

```diff
@@ -32,7 +35,7 @@ When performing a technical evaluation of a module, create a copy of this docume
   * -_note: these tests are defined in https://github.com/folio-org/stripes-testing_
 * [ ] Have i18n support via react-intl and an `en.json` file with English texts
 * [ ] Have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension
-* [ ] Use the latest release of Stripes at the time of evaluation
+* [ ] Use release of Stripes referred to by an [Officially Supported Technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page that is labelled either ACTIVE or ACCEPTED at the time of evaluation
```

Comment:
> the question became which version of OST we point to

Given that a TCR is submitted in the context of an ongoing flower release. I would imagine it would be the OST for the current flower release (up until the submission threshold for that release)

Reply:
This works for me and clarifies that, submissions look forward given they want to be included in a forthcoming release, they should aim for compatibility with that forthcoming release.

Reply:
Let's leave this open until finish cleaning up OST statuses.

Reply:
I'm way late to the party, but this sounds right to me

Reply:
This contradicts the `applies to new modules` column in the [OST docs](https://wiki.folio.org/display/TC/Officially+Supported+Technologies). If we make this change that column should be similarly adjusted

My understanding from that doc is that a release can stay active whilst ever they are supported, meaning that we could have multiple releases in `ACTIVE` status.

According to the example cycle section of that document, by the time a module is submitted then the next release should already be in an `ACCEPTED` status because modules can only be submitted _after_ the scope decomposition deadline for that release

Have I misunderstood the timing of the OST process in relation to the module evaluation process?

My concern is that module developers could choose a version of stripes from a different release which would be incompatible with the release being targeted for inclusion

What are the circumstances where a module should target and `ACTIVE` release rather than `ACCEPTED`? And how should the module developers decide which release to choose?

Similarly, I've just realised there is also a small window where there could be two `ACCEPTED` status release pages, due to the deadline for acceptance by the TC falling between the scope decomposition deadline for the next release and the feature freeze for the current release

Reply:
> This contradicts the applies to new modules column in the OST docs. If we make this change that column should be similarly adjusted

Good catch, thanks.  We have a running "Future Meeting" list of [changes ](https://wiki.folio.org/display/TC/TCR+Process+Improvements+-+Fall+2023) that may have to be made after the PR is merged.

> What are the circumstances where a module should target and ACTIVE release rather than ACCEPTED? And how should the module developers decide which release to choose?

I think @zburke can best answer this (or he may agree that ACCEPTED is all we need to target).  My thought is that per the OST page example, there is a six-week gap between the Quesnelia page going ACTIVE (at Quesnelia feature freeze, 2024-03-15) and Quesnelia going GA (2024-04-29).  Will any teams still be pushing to have their already-developed new modules accepted for the Q release?

Reply:
What I said above is likely incorrect following the conversation on `tc-internal` about differences between the documented OST process and examples that likely threw my thinking off

Reply:
From slack conversation yesterday, I think we have consensus on "the most recent ACTIVE release".  I will work that into the PR.